test/coverage/gen_build.sh

```diff
@@ -67,10 +67,12 @@ EOF
   done
   cat << EOF
     ],
-    tags = ["manual"],
+    # no-remote due to https://github.com/bazelbuild/bazel/issues/4685
+    tags = ["manual", "no-remote"],
     coverage = False,
-    # Needed when invoking external shell tests etc.
-    local = True,
+    # Using the maximum sharding to allow Bazel to schedule them across CPUs.
+    # Sharding can be disabled by --test_sharding_strategy=disabled.
+    shard_count = 50,
```

Comment:
Should this be a function of the number of local CPUs?

Reply:
No, I noted the reason in the comment. Unlike other places that we shard tests, the shards are very uneven, the single test from //test/integration:hotrestart_test takes > 100s while other tests are very small, sharding to maximum allow bazel to schedule the time taking shards run in one core while the other core may run tens shard that only takes ~10s.

Reply:
Can you say this more explicitly in the comment?